File: MQTT_Python/pubApp.py
import paho.mqtt.client as mqtt
# pip install paho-mqtt

client = mqtt.Client()
client.connect("localhost",1883,60)

while True:
    client.publish("LINTANGtopic/test", input('Message: '));
    # client.disconnect();
    # client.loop_forever()

File: MQTT_JavaScript/pubApp.js
var mqtt = require('mqtt');     // npm install mqtt
var client  = mqtt.connect('mqtt://localhost');

var topic = 'LINTANGtopic123'
var message = 'Hello world!'

client.on('connect', function () {
    setInterval(function() {
        client.publish(topic, message);
        console.log('Message sent!', message);
    }, 5000);
});